display_list/display_list_canvas_unittests.cc

```diff
@@ -245,22 +254,21 @@ class BoundsTolerance {
   SkScalar discrete_offset_ = 0;
 };
 
-typedef const std::function<void(SkCanvas*, SkPaint&)> CvSetup;
-typedef const std::function<void(SkCanvas*, const SkPaint&)> CvRenderer;
-typedef const std::function<void(DisplayListBuilder&)> DlRenderer;
-static const CvSetup kEmptyCvSetup = [](SkCanvas*, SkPaint&) {};
-static const CvRenderer kEmptyCvRenderer = [](SkCanvas*, const SkPaint&) {};
-static const DlRenderer kEmptyDlRenderer = [](DisplayListBuilder&) {};
+typedef const std::function<void(SkCanvas*, SkPaint&)> SkSetup;
```

Comment:
I realize this was just renaming existing stuff, but prefer `using` to `typedef`. It usually doesn't matter except you need to use `using` if you need template parameters. So its better (IMO) to be consistent and it seems to be the C++ way.